app/src/main/java/com/duckduckgo/app/onboarding/store/OnboardingStore.kt

```diff
@@ -20,7 +20,9 @@ import com.duckduckgo.app.cta.ui.DaxBubbleCta.DaxDialogIntroOption
 
 interface OnboardingStore {
     var onboardingDialogJourney: String?
+    var visitSiteCtaDisplayCount: Int?
```

Comment:
I usually prefer to include a null value to account for potential scaling needs. However, since this is a temporary property, I agree it makes sense to simply return 0 by default. No need to add extra complexity that could make it more confusing.